recipes/dbus/1.x.x/conanfile.py

```diff
@@ -0,0 +1,108 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = "AFL-2.1"
```

Comment:
```suggestion
    license = ("AFL-2.1", "GPL-2.0-or-later")
```

https://gitlab.freedesktop.org/dbus/dbus/-/blob/dbus-1.12/COPYING#L1